mindsdb_native/libs/helpers/text_helpers.py

```diff
@@ -170,103 +168,58 @@ def isascii(string):
     return all(ord(c) < 128 for c in string)
 
 
-def get_identifier_description(data, column_name, data_subtype, other_potential_subtypes):
-    data = list(data)
+def get_pct_auto_increment(data):
+    data = sorted(data)
+    prev = None
+    increase_by_one = 0
+    for point in data:
+        if prev is not None and point is not None:
+            try:
+                diff = int(point) - int(prev)
```

Comment:
each data point can be convertd to digit-only characters : `int([char if char.isdigit() for char in str(point)])`

this woud look something like this:
```
def to_digits(point):
    return int([char if char.isdigit() for char in str(point)])
...
if prev is not None and to_digits(point) != '':
    diff = int(to_digits(point)) - int(to_digits(prev))
```

Reply:
this would work for data like `abcdef_1, abcdef_2, abcdef_3, ...`

Reply:
Interesting take, I think it should be something else other than "auto incrementing index" though, since it's not that same thing, but we should add this approach.